### PR TITLE
Correct French spelling in the manifest

### PR DESCRIPTION
**Pull request by Linkid**
## Problem
- *In the manifest, there are some spelling errors in the French description*

## Solution
- *French description corrected*

## PR Status
- [X] Code finished.
- [ ] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [X] Upgrade from last version tested.
- [X] Can be reviewed and tested.

## Validation
---
*Minor decision*
- **Upgrade previous version** : 
- [ ] **Code review** : 
- [ ] **Approval (LGTM)** : 
- [ ] **Approval (LGTM)** : 
- **CI succeeded** : 
[Build Status](https://ci-apps-hq.yunohost.org/jenkins/job/strut_ynh%20PR-NUM-/)  
*Please replace '-NUM-' in this link by the PR number.*  
When the PR is marked as ready to merge, you have to wait for 3 days before really merging it.

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     },
     "description": {
         "en": "Strut is a slide editor creating impress.js presentations.",
-        "fr": "Strut est un éditeur de slide permetant de créer des présentation impress.js."
+        "fr": "Strut est un éditeur de diapositives permettant de créer des présentations impress.js."
```
Comment (ericgaspar) [inline manifest.json:13]:
```suggestion
        "fr": "Éditeur de diapositives permettant de créer des présentations impress.js."
```

File: manifest.json [hunk 2]
```diff
@@ -10,7 +10,7 @@
     },
     "description": {
         "en": "Strut is a slide editor creating impress.js presentations.",
```
Comment (ericgaspar) [inline manifest.json:12]:
```suggestion
        "en": "Slide editor for creating impress.js presentations.",
```

File: manifest.json (hunk 1 above)
Comment (Linkid) [reply manifest.json]: Thanks for the suggestion :)

Comment (Linkid) [reply manifest.json]: Thanks for the suggestion :)

File: manifest.json (hunk 2 above)
Comment (Linkid) [reply manifest.json]: Thanks for the suggestion :)

**Review by zamentur — APPROVED** [review]

**Pull request merged**